producers/http/handlers.go

```diff
@@ -25,6 +25,23 @@ import (
 	"github.com/gorilla/mux"
 )
 
+func debugHandler(p *producerImpl) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		allMetrics := p.store.Objects()
+		if len(allMetrics) == 0 {
+			httpLog.Error("/v0/debug - no content in store.")
```

Comment:
Is there a reason the handlers in this file hardcode their paths in error message instead of getting them from `r.URL.Path`?

Reply:
they should definitely do that